test/libsolidity/SolidityABIJSON.cpp

```diff
@@ -499,7 +499,8 @@ BOOST_AUTO_TEST_CASE(constructor_abi)
 {
 	char const* sourceCode = R"(
 		contract test {
-			function test(uint param1, test param2, bool param3) {}
+			enum ActionChoices { GoLeft, GoRight, GoStraight, Sit }
```

Comment:
Was this unit test wrong? If not, I would recommend to keep it, and another one (e.g. "constructor_abi2").